src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,17 @@
 package core.basesyntax;
 
+import core.basesyntax.controller.ConsoleHandler;
+import core.basesyntax.lib.Injector;
+import java.lang.reflect.InvocationTargetException;
+
 /**
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
```

Comment:
Not informative naming. Since you've decided to leave the class, change its name via refactoring.

Reply:
corrected